libs/assets/batch/startup.bat

```diff
@@ -0,0 +1,9 @@
+sc query "PM2" | find "RUNNING"
```

Comment:
This will only work IF this script is running. 

The windows service process named "PM2" created by NSSM  is *this* script. It will not check if the actual pm2 process is running or not.

Reply:
Addressed per discussion